src/main/java/org/owasp/validator/css/CssHandler.java

```diff
@@ -120,16 +120,13 @@
 	 * 
 	 * @param policy
 	 *            the policy to use
-	 * @param embeddedStyleSheets
```

Comment:
So apparently this parameter isn't needed anymore? Even if that is true, removing it from the API is an API breaking change. Instead, can you create a new API that does NOT take this parameter, and then mark the old API deprecated.  We can't simply add/remove parameters from public methods without going through a deprecation process first, as we don't want to break the API.

Reply:
On it. Of course you will have the spotbug issue we're trying to remove by leaving the old API.
I'm reverting the changes and adding new constructors with the removed parameter.